File: TexasHoldemAI(足够筹码不会弃牌版)/TexasHoldemAI/TexasHoldemAI/OnHand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TexasHoldemAI
{
    class OnHand
    {
        Porker[] porkers;
        public OnHand(Porker[] porkers)
        {
            this.Porkers = porkers;
        }

        public Porker[] Porkers { get { return porkers; } set { porkers = value; } }
    }
}

File: C++版本/OnHand/OnHand.cpp
#include "D:\c++学习\新建文件夹 (2)\计算机博弈大赛伯懿德州扑克\C++版本\OnHand\OnHand.h"
#include "D:\c++学习\新建文件夹 (2)\计算机博弈大赛伯懿德州扑克\C++版本\Poker\Poker.h"

OnHand::OnHand(const std::vector<Poker>& porkers) {
    this->porkers = porkers;
}

const std::vector<Poker>& OnHand::getPorkers() const {
    return porkers;
}

void OnHand::setPorkers(const std::vector<Poker>& porkers) {
    this->porkers = porkers;
}